modules/administration/pages/troubleshooting/tshoot-autoinstallation.adoc

```diff
@@ -1,17 +1,11 @@
 [[troubleshooting-autoinstallation]]
 = Troubleshooting Autoinstallation
 
-Depending on your base channel, new autoinstallation profiles might be subscribed to a channel that is missing required packages.
+If you use the obsolete plain {salt} (salt-minion) implementation for communication, you on your own must make sure that all the needed software (software packages) is properly available in configured client channels.
 
-For autoinstallation to work, these packages are required:
+With the default {salt} bundle (venv-salt-minion) implementation you only need a client tools channel as a child channel that is compatible with your client base channel.
+All required packages will be part of the {salt} bundle.
 
-* [package]``pyOpenSSL``
-* [package]``rhnlib``
-* [package]``libxml2-python``
-* [package]``spacewalk-koan``
-
-To resolve this issue, check these things first:
-
-* Check that the tools software channel related to the base channel in your autoinstallation profile is available to your organization and your user.
+* Check that the client tools software channel related to the base channel in your autoinstallation profile is available to your organization and your user.
 * Check that the tools channel is available to your {productname} as a child channel.
```

Comment:
Are these 2 maybe the same?

Reply:
yes, I think it is the same.  I think with the first item we want to say the organization and the user must have permissions to use it and with the second that the Manager must have it available/ready to be used.  The second might be superfluous or, maybe, it should come first.